src/Caliburn.Micro.Core/Screen.cs

```diff
@@ -206,6 +227,15 @@ protected virtual Task OnActivateAsync(CancellationToken cancellationToken)
         }
 
 
+        /// <summary>
+        /// Called when exception called in OnActivatedAsync.
+        /// </summary>
+        protected virtual void OnActivatedAsyncException(Exception thrownException)
```

Comment:
This gets called when `OnActivateAsync` or `OnActivatedAsync` throws an exception but the name `OnActivatedAsyncException` suggests it is related to the `OnActivatedAsync` method. Maybe we should split this into two methods? Like `OnActivateAsyncException` and `OnActivatedAsyncException`? Any other ideas?

PS: Same for `OnInitializedAsyncException`

Reply:
I think the rename from OnActivateAsync to OnActivatedAsync should be reverted, since that function is called when the framework wants to activate the screen and not when it is already activated.